src/Bridge/Doctrine/Orm/Paginator.php

```diff
@@ -14,95 +14,32 @@
 namespace ApiPlatform\Core\Bridge\Doctrine\Orm;
 
 use ApiPlatform\Core\DataProvider\PaginatorInterface;
-use Doctrine\ORM\Tools\Pagination\Paginator as DoctrineOrmPaginator;
 
 /**
  * Decorates the Doctrine ORM paginator.
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-final class Paginator implements \IteratorAggregate, PaginatorInterface
+final class Paginator extends AbstractPaginator implements PaginatorInterface
```

Comment:
`implements PaginatorInterface` can be removed (already implemented by the `abstract` class).

Reply:
Nop, `AbstractPaginator` implements `PartialPaginatorInterface`.

Reply:
Not keep it as is :) Sorry.